### PR TITLE
[doc](typo) external-table-load.md

### PR DESCRIPTION
**Pull request by JoverZhang**
The SQL with a syntax error.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [x] Is this PR support rollback (If NO, please explain WHY)

### CODE REVIEW

File: docs/en/docs/data-operate/import/import-scenes/external-table-load.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Here we create an `ext_oracle_demo` external table and reference the `oracle_tes
      `k6` double NOT NULL COMMENT ""
    )
    COMMENT "Doris Table"
-   DISTRIBUTED BY HASH(k1) BUCKETS 2;
+   DISTRIBUTED BY HASH(k1) BUCKETS 2
```
Comment (carlvinhust2012) [inline docs/en/docs/data-operate/import/import-scenes/external-table-load.md:104]: Does the Chinese version no problem?

**Comment by yagagagaga** [comment]
docs/zh-CN/docs/data-operate/import/import-scenes/external-table-load.md

have the same problem. Please modify it by the way.

**Comment by JoverZhang** [comment]
> docs/zh-CN/docs/data-operate/import/import-scenes/external-table-load.md
> 
> have the same problem. Please modify it by the way.

OK, I changed it by the way.

**Review by luzhijing — APPROVED** [review]
LGTM

**Comment by luzhijing** [comment]
run build all

**Comment by luzhijing** [comment]
run P0

**Pull request merged**